src/services/item/plugins/itemLike/itemLike.schemas.ts

```diff
@@ -60,6 +60,22 @@ export const getLikesForItem = {
   },
 };
 
+const rawItemLikeSchemaRef = registerSchemaAsRef(
+  'rawItemLike',
+  'Raw Item Like',
+  customType.StrictObject(
+    {
+      id: customType.UUID(),
+      itemId: customType.UUID(),
+      creatorId: customType.UUID(),
+      createdAt: customType.DateTime(),
+    },
+    {
+      description: 'Raw Like entry created when a member likes a published collection.',
```

Comment:
Should we mention "published"? I think the backend does not prevent the like if the item is not published.